workbench-core/swb-common-ui/src/common/tableEmptyState.tsx

```diff
@@ -4,6 +4,7 @@
  */
 
 import { Box, Button } from '@cloudscape-design/components';
+import * as React from 'react';
```

Comment:
This import is necessary to have the jsx compile, correct?

Reply:
Correct